.prettier.config.js

```diff
@@ -1,35 +1,35 @@
-'use strict';
+"use strict";
 
 module.exports = {
-  arrowParens: 'always',
+  arrowParens: "always",
```

Comment:
Codacy has a fix for the issue: [Replace `"always"` with `'always'`](https://app.codacy.com/gh/openinf/openinf.github.io/pullRequest?prid=9929905)
```suggestion
  arrowParens: 'always',
```